istio/istio.go

```diff
@@ -424,6 +446,37 @@ func (iClient *IstioClient) ApplyOperation(ctx context.Context, arReq *meshes.Ap
 		return &meshes.ApplyRuleResponse{
 			OperationId: arReq.OperationId,
 		}, nil
+	case installHipsterShopCommand:
+        go func() {
+            opName1 := "deploying"
+            if arReq.DeleteOp {
+                opName1 = "removing"
+            }
+            if err := iClient.executeHipsterShopInstall(ctx, arReq); err != nil {
+                iClient.eventChan <- &meshes.EventsResponse{
+                    OperationId: arReq.OperationId,
+                    EventType:   meshes.EventType_ERROR,
+                    Summary:     fmt.Sprintf("Error while %s the Hipster Shop", opName1),
```

Comment:
please change Hipster shop to something like google micro services demo application.

Reply:
Just seeing this... yes, @girishranganathan, great suggestion. +1 for "Hipster Shop".

Reply:
cool, if we are ok with it 👍